### PR TITLE
#10-task-1

### PR DESCRIPTION
**Pull request by JPerzanowski**

### CODE REVIEW

File: src/com/javafee/tasks/task1/Main.java [hunk 1]
```diff
@@ -171,38 +171,60 @@ public static void task9() {
     // ...
     public static void task10() {
         int num = new Scanner(System.in).nextInt();
-        int last = 0;
-        int lastMinusOne = 0;
+        int count = 0;
+        int[] reverse = new int[num];
```
Comment (JaRutkowski) [inline src/com/javafee/tasks/task1/Main.java:175]: Could u please try to do it without using array? :)

File: src/com/javafee/tasks/task1/Main.java [hunk 2]
```diff
@@ -171,38 +171,60 @@ public static void task9() {
...
-                    System.out.print(x + w + " ");
-                    x--;
-
-                }
+            for (int i = reverse.length; i >= 1; i--)
+                System.out.print(reverse[i - 1] + " ");
             System.out.println();
         }
     }
+//        int num = new Scanner(System.in).nextInt();
```
Comment (JaRutkowski) [inline src/com/javafee/tasks/task1/Main.java:195]: What is that code for?

**Review by JaRutkowski — CHANGES_REQUESTED** [review]
PSB (comments)

File: src/com/javafee/tasks/task1/Main.java (hunk 1 above)
Comment (JPerzanowski) [reply src/com/javafee/tasks/task1/Main.java]: I did it

File: src/com/javafee/tasks/task1/Main.java (hunk 2 above)
Comment (JPerzanowski) [reply src/com/javafee/tasks/task1/Main.java]: It shows an amount of numbers in a row

Comment (JaRutkowski) [reply src/com/javafee/tasks/task1/Main.java]: Is it needed - as it is the comment only? IMO all comments should be removed.

Comment (JPerzanowski) [reply src/com/javafee/tasks/task1/Main.java]: Sorry, I didn't notice it's a comment.

File: src/com/javafee/tasks/task1/Main.java [hunk 3]
```diff
@@ -171,37 +171,29 @@ public static void task9() {
...
-                    p++;
-                }
-            }
-            System.out.println();
-            for (int i = 1, k = last, x = lastMinusOne, p = counter - 1; i <= num; i++)
-                if (i % 2 != 0) {
-                    System.out.print(k + w + " ");
-                    k -= p;
-                    p--;
+            for (int i = 1, k = 2 + count, x = 4 + count, p = 1; i <= (2 * num) + 1; i++)
```
Comment (JaRutkowski) [inline src/com/javafee/tasks/task1/Main.java:176]: I'm sure that it's possible to make it without such many if instructions, please try to do it in a more simple way (without many if's).

**Review by JaRutkowski — CHANGES_REQUESTED** [review]
PSB (comments)

File: src/com/javafee/tasks/task1/Main.java (hunk 3 above)
Comment (JaRutkowski) [reply src/com/javafee/tasks/task1/Main.java:176]: I'll try to give u some examples, maybe not fully done, but something that will help u in the further implementation.

Comment (JaRutkowski) [reply src/com/javafee/tasks/task1/Main.java:176]: Did u have a chance to look at it?

**Review by JaRutkowski — CHANGES_REQUESTED** [review]
Line 57 - unnecessary brackets. 
Line 147 - is it possible to move k/p to loop and then use the ternary operator?
Line 154 - unnecessary comment.
// Task 10 - what is the status?

File: src/com/javafee/tasks/task1/Main.java (hunk 3 above)
Comment (JPerzanowski) [reply src/com/javafee/tasks/task1/Main.java:176]: I will do it right now.

**Review by JaRutkowski — APPROVED** [review]
Please merge and create the 'bug' type issues for the improvements.

**Pull request merged**